processor/src/main/resources/org/mapstruct/ap/internal/model/BeanMappingMethod.ftl

```diff
@@ -76,9 +76,12 @@
     	</#if>
     	<@includeModel object=callback targetBeanName=resultName targetType=resultType/>
     </#list>
-    <#if returnType.name != "void">
```

Comment:
This part here is not connected with `@AfterMapping`, `@AfterMapping` occurs before this. This part here generates the return of the method. 

Yes I am saying that it should be like the second example. The reason for this is that if the return type of the method for example:

```
void update(Source source, @MappingTarget Target target);
```

If for this method the `returnType` is `void`, but the `resultType` is `Target`. If we do like it is now and there is a builder for the `resultType` then there will be a compilation error.